packages/fast-tooling-react/README.md

```diff
@@ -240,3 +241,126 @@ const childOptions = [
 
 const data = getDataFromSchema(schema, childOptions);
 ```
+
+## Navigation
+
+The Navigation uses data structures as specified by the [children](#data-structures-for-react-children) data structure to create a treeview. It includes a callback for updating the data location if you are using it with other exports from this package and can be controlled or uncontrolled.
```

Comment:
A few nits:

- I see `tree view` written in different ways (`tree view`, `treeview`, `tree-view`). Can we be consistent with how we write it (I think `tree view` is the most correct)?
- `Navigation` shouldn't be capitalized.